admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -0,0 +1,193 @@
+==========================
+Configuring Memory Caching
+==========================
+
+You can significantly improve your ownCloud server performance with memory 
+caching, where frequently-requested objects are stored in memory for faster 
+retrieval. There are two types of caches to use: PHP opcache, and data caching 
```

Comment:
I still say that it should be called "opcode cache" instead of using the term opcache which gets mixed with the PHP module called "OPCache".